grafana-plugin/src/containers/DefaultPageLayout/DefaultPageLayout.helpers.tsx

```diff
@@ -10,12 +11,15 @@ export function getSlackMessage(slackError: SlackError, organization: Organizati
     return (
       <>
         Couldn't connect Slack.
-        {Boolean(organization?.slack_team_identity) && (
-          <>
-            {' '}
-            Select <b>{organization.slack_team_identity.cached_name}</b> workspace when connecting please
-          </>
-        )}
+        <RenderConditionally
+          shouldRender={Boolean(organization?.slack_team_identity)}
+          render={() => (
+            <>
+              {' '}
+              Select <b>{organization.slack_team_identity.cached_name}</b> workspace when connecting please
+            </>
+          )}
+        />
```

Comment:
This change doesn't harm but I think it doesn't change much. The problem with `slack_team_identity` comes from Users.tsx line 363
![image](https://github.com/grafana/oncall/assets/12073649/b61f3a3f-daeb-44ad-a00b-ef038bade4d6)

Reply:
Ah you changed there too, nvm

Reply:
I see you changed there too, nvm